python_modules/libraries/dagster-gcp/dagster_gcp_tests/dataproc_tests/test_resources.py

```diff
@@ -22,6 +22,7 @@
 DATAPROC_CLUSTERS_URI = f"{DATAPROC_BASE_URI}/clusters"
 DATAPROC_JOBS_URI = f"{DATAPROC_BASE_URI}/jobs"
 DATAPROC_SCHEMA_URI = "https://www.googleapis.com/discovery/v1/apis/dataproc/v1/rest"
+DATAPROC_LABELS = {"test": "true"}
```

Comment:
can you add a second label to this dict?

Reply:
sure, added another label